benchmarks/src/test/java/org/apache/druid/benchmark/query/SqlBaseBenchmark.java

```diff
@@ -355,6 +368,11 @@ public void setup() throws JsonProcessingException
 
   private void checkIncompatibleParameters()
   {
+    // we only support NONE object storage encoding for auto schema with mmap segments
+    if (ObjectStorageEncoding.NONE.equals(objectStorageEncoding) && !("auto".equals(schemaType)
```

Comment:
just checking for auto is maybe not quite correct, like we definitely want to run them if 'auto' is set, but a non-auto benchmark dataset could also have a json column that explicitly defines it as nested/auto too.

I think we want something like this inside this if
```
for (String dataSource : getDatasources()) {
      final DimensionsSpec dims = SqlBenchmarkDatasets.getSchema(dataSource).getDimensionsSpec();
      if (dims.getDimensions().stream().noneMatch(x -> x instanceof AutoTypeColumnSchema || x instanceof NestedDataColumnSchema)) {
        System.exit(0);
      }
    }
```